### ISSUE TITLE
Threading issue in LottieTask

### ISSUE THREAD

**Issue by elihart** [airbnb/lottie-android#930]
Crash
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void java.lang.Thread.interrupt()' on a null object reference
        at com.airbnb.lottie.LottieTask.stopTaskObserverIfNeeded(LottieTask.java:215)
        at com.airbnb.lottie.LottieTask.access$500(LottieTask.java:28)
        at com.airbnb.lottie.LottieTask$2.run(LottieTask.java:198)
```

Looks like on a single thread this can't happen as the thread is checked for null a few lines before:
https://github.com/airbnb/lottie-android/blame/master/lottie/src/main/java/com/airbnb/lottie/LottieTask.java#L216

So it seems that access to the thread isn't properly synchronized

**Comment by gpeal** [comment]
Thanks! I'll publish a dot update

**Comment by Jazeem** [comment]
I'm getting the same error even with the synchronized method update. @elihart can you comment whether it was fixed for you with the latest version?

![screen shot 2018-09-19 at 11 56 26 pm](https://user-images.githubusercontent.com/7645229/45773370-c3f3fa80-bc67-11e8-8df6-9ee712096818.png)

**Comment by elihart** [comment]
Just want to say that this isn't happening at all anymore, so the fix seems to work great

**Comment by canyara** [comment]
still happen on lottie-react-native 2.5.11 which is "com.airbnb.android:lottie:2.6.0'"

**Comment by gpeal** [comment]
@canyara This was fixed in 2.6.1
